apps/browser-extension-wallet/src/views/browser-view/features/wallet-setup/helpers.ts

```diff
@@ -1,5 +1,33 @@
+/* eslint-disable camelcase */
 import { Wallet } from '@lace/cardano';
+import { LedgerKeyAgent } from '@cardano-sdk/hardware-ledger';
 
 export const isTrezorHWSupported = (): boolean => process.env.USE_TREZOR_HW === 'true';
 export const isHardwareWalletAvailable = (wallet: Wallet.HardwareWallets): boolean =>
   wallet !== Wallet.KeyManagement.KeyAgentType.Trezor || isTrezorHWSupported();
+type HardwareWalletPersonaProperties = {
```

Comment:
Small suggestion - PostHog calls those Person Properties, let's use the same vocabulary everywhere :) 
```suggestion
type HardwareWalletPersonProperties = {
```